selfdrive/car/hyundai/values.py

```diff
@@ -1883,9 +1883,11 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
   CAR.KIA_CARNIVAL_4TH_GEN: {
     (Ecu.fwdCamera, 0x7c4, None): [
       b'\xf1\x00KA4 MFC  AT USA LHD 1.00 1.06 99210-R0000 220221',
+      b'\xf1\x00KA4CMFC  AT CHN LHD 1.00 1.01 99211-I4000 210525',
```

Comment:
real quick can you check if the chinese version is meaningfully different? like in wheelbase, steer ratio, weight, etc

Reply:
> real quick can you check if the chinese version is meaningfully different? like in wheelbase, steer ratio, weight, etc

I initially checked when the first PR was opened and those items were identical among the matching trims.